package.js

```diff
@@ -1,7 +1,7 @@
 /* jshint ignore:start */
 Package.describe({
     name: 'ethereum:web3',
-    version: '1.2.1',
+    version: '1.2.2',
```

Comment:
Will this number get bumped automatically during the release process, or should it be `1.2.3`?